front/src/containers/Notification/style.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+import { StyleSheet } from 'react-native'
+
+export const GlobalStyle = StyleSheet.create({
+  Icon: {
+    zIndex: 1,
+    position: 'absolute',
+    left: 65,
```

Comment:
ohh, this it's right? because in a smaller or biggest cell phone, this position will be correct?